src/databricks/labs/ucx/install.py

```diff
@@ -529,6 +530,16 @@ def _create_database(self):
                     "UCX Install: databricks labs install ucx"
                 )
                 raise BadRequest(msg) from err
+            if "Unable to load AWS credentials from any provider in the chain" in str(err):
```

Comment:
Would prefer to use `error_code`:  https://github.com/databricks/databricks-sdk-py/blob/fb30ed98912a4e62a4b535188765f792b7bc6f47/databricks/sdk/errors/base.py#L36